application-vanilla/src/commonMain/kotlin/com/fraktalio/fmodel/application/EphemeralViewExtension.kt

```diff
@@ -0,0 +1,14 @@
+package com.fraktalio.fmodel.application
+
+import kotlinx.coroutines.flow.fold
+
+/**
+ * Extension function - Handles the query of type [Q]
+ *
+ * @param query Query of type [Q] to be handled
+ * @return State of type [S]
+ *
+ * @author Domenic Cassisi
+ */
+suspend fun <S, E, Q, MV> MV.handle(query: Q): S where MV : ViewStateComputation<S, E>, MV : EphemeralViewRepository<E, Q> =
```

Comment:
MV stands for MaterializedView. Should we consider EV for this case?

Reply:
Done, thanks!